tools/harness-thci/OpenThread.py

```diff
@@ -2720,7 +2725,12 @@ def setMinDelayTimer(self, iSeconds):
 
     @API
     def ValidateDeviceFirmware(self):
-        return 'OPENTHREAD' in self.UIStatusMsg
+        if self.DeviceCapability == DevCapb.V1_1:
+            return OT11_VERSION in self.UIStatusMsg
+        elif self.DeviceCapability == (DevCapb.L_AIO | DevCapb.C_FFD | DevCapb.C_RFD):
```

Comment:
Are we sure the a Thread 1.2 device must have this exact `DeviceCapability`?

Reply:
This is what we set in `__discoverDeviceCapability`. Do you think a global variable used in both places would help to keep consistency?